Service/AddOn/ReadmeBuilder.php

```diff
@@ -819,7 +819,7 @@ protected function _save() : void
 
                 $htmlConverter = new HtmlConverter($environment);
                 $htmlConverter->getConfig()->setOption('suppress_errors', true);
-                $contents = $htmlConverter->convert($readmeHtml);
+                $contents = str_replace('  \n', '<br />', $htmlConverter->convert($readmeHtml));
```

Comment:
It would be a lot better if the place where header and column are set does the eol => br translation. Thonks?

Reply:
No, because the HTML to Markdown converter converts `<br>` tags to `  \n` (two spaces then a newline character). The conversion has to happen _after_ the HTML Converter is ran.